articles/quickstart/webapp/any/01-login.md

```diff
@@ -0,0 +1,178 @@
+
+1. Your application redirects the user's browser to Auth0 `/authorize` endpoint, with a *redirect URI* in the query string.
+2. The user logs in, using the Auth0 login form on that page.
+3. Auth0 authenticates the user. 
+4. Auth0 redirects the user to your `redirect_uri`, with an *Authorization Code* in the querystring.
+5. Your application calls the Auth0 /oauth/token endpoint with the Authorization Code, asking to exchange it with an access_token.
+6. Auth0 authenticates the web app, validates the Authorization Code, and responds back with the token.
+7. The web application decodes and verifies the token, which confirms the user's identity.
+8. Your application uses sessions or cookies to persist a user login.
+
+
+### About User Authentication
+
+When a user arrives at the Auth0 authentication endpoint, they can log in with a third-party service (such as Google, Facebook, or Github) or with email and password. Email and password logins are authenticated against a database of users stored with Auth0 and associated to your account. 
+
+You can import existing user information in bulk or add users manually in the Auth0 dashboard. Users may also *sign up* for your application with their email address, which Auth0 will verify via confirmation email.
+
+In any case, Auth0 provides back to your application the authenticated identity of the user --- a unique third-party profile or email address --- and it is up to your application to respond to that identity apporopriately. 
+
+## Register Your App as a Client
+
+Go to the [Clients](${manage_url}/#/clients) page from the [Auth0 Dashboard](${manage_url}). Edit the default client or create a new client by clicking **Create Client**.
+
+Name your new application, and set the Client Type to **Regular Web Applications** as the **Client Type**.
+
+### Set Allowed Callback URI
+
+When your app redirects a user to Auth0, the querystring will include a callback URI. After authentication completes, the user will be redirected to this URI.
+
+${include('../_callbackRegularWebApp')}
+
+While you are editing your Client settings, update your Allowed Callbacks list to include the redirect URI you plan to use.
+
+### Enable OIDC Conformance
+
+Before you leave Client Setting, click on **Show Advanced Settings**. Go to the **OAuth** tab and make sure that the **OIDC Conformant** switch is enabled.
+
+Save your settings.
+
+## Create a Log In Link
+
+Your users will log in to your site using the Auth0 authentication portal at the URI `https://${account.namespace}/authorize`.
+
+To facilitate this, you will provide a link from your site with the following query parameters:
+
+| Parameter | Value |
+|:----------|:---------|
+| response_type | `code` |
+| client_id | ${account.clientId} |
+| scope | `openid` |
+| redirect_uri | The URI that the user will be redirected to after authentication (`${account.callback}`)|
```

Comment:
Are there other possible values for scope that make sense for this web application scenario?

Reply:
Yes, but I was trying to keep this quick start as simple as possible, instead of getting into every option.